lib/transforms/onnxextension_legalizer.cc

```diff
@@ -660,13 +660,13 @@ static std::vector<Def> ConvertSplit(const ONNXExtensionInst* ext,
   int input_dims = input_type.GetNumOfDims();
 
   HLCHECK(ext->GetNumOfAttributes() == 2);
-  const Attribute* attr = ext->GetAttributes()[0].get();
+  const Attribute* attr = ext->GetAttributes()[1].get();
```

Comment:
we can use "FindAttribute(*ext, "axis", 0)"

Reply:
less error-prone

Reply:
ok